model_signing/README.md

```diff
@@ -1,6 +1,22 @@
 # Model Signing
 
-This project demonstrates how to protect the integrity of a model by signing it with [Sigstore](https://www.sigstore.dev/).
+This project demonstrates how to protect the integrity of a model by signing it
+with [Sigstore](https://www.sigstore.dev/), a tool for making code signatures
+transparent without requiring management of cryptographic key material.
+
+When users download a given version of a signed model they can check that the
+signature comes from a known or trusted identity and thus that the model hasn't
+been tampered with after training.
+
+Signing events are recorded to Sigstore's append-only transparency log.
+Transparency logs make signing events discoverable, so that model signers can
```

Comment:
Right now I'm not sure it's clear what discoverability means, or how it's tied to the inclusion proof.  I'd propose changing these two sentences to something like: "Transparency logs make signing events discoverable, meaning model verifiers can ensure that the models they see are in the log (by checking a proof of inclusion, which is handled by the model signing library) and thus will eventually be seen by model signers that monitor the log to check for any unexpended signing events.

Reply:
Thank you, incorporated the suggestion